packages/lesswrong/lib/cookies/cookies.ts

```diff
@@ -176,6 +176,12 @@ export const HIDE_EAG_BANNER_COOKIE = registerCookie({
   description: "Don't show any EAG(x) banners",
 });
 
+export const HIDE_EA_FORUM_SURVEY_BANNER_COOKIE = registerCookie({
+  name: "hide_ea_forum_survey_banner",
```

Comment:
It might be wise to include a year in this cookie name - I can't remember what the cookie name was last year, but we probably don't want to accidentally hide the new survey if people hid the old one (or hide next years if they hide this one).

Reply:
Thanks! The cookie is set to expire in 3 months, so I think it should be fine to reuse once a year?